js/core/utils/resize_callbacks.js

```diff
@@ -15,10 +15,10 @@ var resizeCallbacks = (function() {
     }
 
     var formatSize = function() {
-        var documentElement = domAdapter.getDocumentElement();
+        var window = windowUtils.getWindow();
         return {
-            width: documentElement.clientWidth,
-            height: documentElement.clientHeight
+            width: window.innerWidth,
```

Comment:
@nightskylark these changes look good to you?

Reply:
I recommend to check it on different jquery versions.

Reply:
And I'm not sure it's well covered by tests. so it would be great to ensure it works fine manually.

Reply:
I have check most common cases with some jquery versions (3.4.1, 3.1.0, 2.2.1, 2.1.2) on android and ios devices (iphone 10), desktop chrome, ie, macOS safari and do not find any regressions.